xcp_d/workflows/base.py

```diff
@@ -478,6 +479,7 @@ def init_single_subject_wf(subject_id: str):
                 name=f'postprocess_{run_counter}_wf',
             )
             run_counter += 1
```

Comment:
do we still need run_counter?

Reply:
Currently it's just used to raise an error if none of the runs in the dataset are postprocessable.